Sources/AutorestSwift/Schema.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-public struct Schema: CodeModelProperty {
+public struct Schema: CodeModelProperty, GroupSchemaAllOf {
```

Comment:
`GroupSchemaAllOf` shouldn't be part of Schema?

Reply:
Added a Todo to revisit this after clarification with Azure team.